src/lib.rs

```diff
@@ -61,6 +61,18 @@ pub fn fast_process_instruction(
         }
     };
 
+    #[cfg(feature = "unit_test_config")]
+    if matches!(
+        discriminator,
+        DlpDiscriminator::Delegate
+            | DlpDiscriminator::CommitState
+            | DlpDiscriminator::CommitStateFromBuffer
+            | DlpDiscriminator::Finalize
+            | DlpDiscriminator::Undelegate
+    ) {
+        msg!("Processing instruction: {:?}", discriminator);
```

Comment:
This change is unrelated to the PR. As mentioned in PR https://github.com/magicblock-labs/delegation-program/pull/109 I suggest to move all logs under a `logging` feature and to simplify it: 

```rust
#[cfg(feature = "logging")]
msg!("Processing instruction: {:?}", discriminator);
```